### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by corburn**

### CODE REVIEW

File: book/fundamentals/phylogeny-reconstruction.md [hunk 1]
```diff
@@ -566,4 +566,4 @@ The material in this section was compiled while consulting the following sources
 
 1. The Phylogenetic Handbook (Lemey, Salemi, Vandamme)
 2. Inferring Phylogeny (Felsenstein)
-3. [Richard Edwards's teaching website](http://www.southampton.ac.uk/~re1u06/teaching/upgma/)
+3. [Richard Edwards' teaching website](http://www.southampton.ac.uk/~re1u06/teaching/upgma/)
```
Comment (gregcaporaso) [inline book/fundamentals/phylogeny-reconstruction.md]: @corburn, I just noticed this. The way I had it was correct here - see [this page](http://www.thepunctuationguide.com/apostrophe.html) for example (a relevant example on that page is *Tom Jones’s first album*).

Could you correct this before I merge?

**Comment by corburn** [comment]
It is corrected.

**Comment by gregcaporaso** [comment]
Thanks! I'm just going to wait for the Travis tests to complete and then this is ready for merge.

**Comment by corburn** [comment]
[database-searching.md](https://github.com/caporaso-lab/An-Introduction-To-Applied-Bioinformatics/blame/6e0d1eb12bb2f81665dbeac5a6cc759cb2c99a59/book/fundamentals/database-searching.md#L616) contains a paragraph with the phrase:

> the sequences we alignment are homologous.

I added a commit to rephrase this and the surrounding paragraph. It is included in this pull request to avoid a potential merge conflict since it affects a line that contained a typo.

**Comment by gregcaporaso** [comment]
Thanks @corburn! I'm going to merge this one since it's only touching text and not touching code. I'm going to leave your other PR (that does touch code) open until I get Travis builds working again.

**Pull request merged**